sql1.txt

```diff
@@ -6,24 +6,28 @@ SELECT * FROM Products LIMIT 5;
 1. Use an SQL SELECT statement to retrieve the first 10 rows of the Customer table, ordered by CustomerName.
 Paste your SQL statement below:
 
+SELECT * FROM Сustomers ORDER BY СustomerName LIMIT 10;
 
 2. Use an SQL Select statement to retrieve the names and prices for all products that cost less than $20.
 Paste your SQL statement below:
 
+SELECT ProductName, Price FROM Products WHERE price < 20;
 
 3. Retrieve all employees whose last name starts with C.  Paste your SQL statement below:
 
+SELECT lastname FROM employees WHERE lastname LIKE "C%";
```

Comment:
Your query is almost there! Here are a couple of adjustments to make it perfect:

1. SELECT Clause: You want to retrieve all the employee information, not just the last names. To do this, replace the first `LastName` with `*` to select all columns.

2. Capitalization: Ensure the table name `Employees` and the column name `LastName` toward the end are correctly capitalized, as you did in your other answers. 

Great job on getting the syntax right!

Reply:
done!